docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Set up & load data for Model Builder
+description: Learn how to set up and load data from a SQL Server database or a file for use in one of the Model Builder scenarios for ML.NET.
+ms.date: 10/31/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc, how-to
+#Customer intent: As a developer, I want to load data in Model Builder
+---
+
+# Load data in Model Builder
+
+Learn how to set up and load your data sets from a SQL Server or a file database for use in one of the Model Builder scenarios for ML.NET. Model Builder scenarios can use SQL files, image files, and CSV or TSV file formats.
+
+## Data set limitations in Model Builder
+
+Model Builder limits the amount of data you can use for training models:
+
+- Training data set limit: 1GB
```

Comment:
```suggestion
```

The 1GB limit is no longer a limitation so it can be removed. 

https://github.com/dotnet/machinelearning-samples/commit/9829c1de3f4b4f8f4fe2e092c8ba03dda16efcf4